py/server/deephaven/_table_reader.py

```diff
@@ -47,7 +47,7 @@ def _table_reader_rows(table: Table, cols: Optional[Union[str, Sequence[str]]])
                 col_dict[col_def.name] = chunk_dict[col_def.name][i]
             yield col_dict
 
-def _table_reader_chunks(table: Table, *, cols: Optional[Union[str, Sequence[str]]], row_set: jpy.JType, chunk_size: Optional[int],
+def _table_reader_chunks(table: Table, *, cols: Optional[Union[str, Sequence[str]]], row_set: jpy.JType, chunk_size: Optional[int] = 4096,
```

Comment:
You still need to change the implementation; if the caller explicitly sets to None, we get `row_set.size()` which is not appropriate.